python/tests/test_parse.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import io
+
+from pyoxigraph import *
+
+
+class TestParse(unittest.TestCase):
+    def test_parse(self):
+        input = io.BytesIO(b'<foo> <p> "1" .')
+        result = list(parse(input, "text/turtle", base_iri="http://example.com/"))
+
+        assert result == [
```

Comment:
I believe it's better to use `self.assertEqual` insert of `assert` in order to get a better integration with `unittest`.

Reply:
Ah yes, thank you. I'm too used to writing pytest tests which uses `assert`, oops.

Reply:
I've moved the tests into a test_io file as well.